tests/lang_es_test.rs

```diff
@@ -0,0 +1,88 @@
+use num2words::{Currency, Lang, Num2Err, Num2Words};
```

Comment:
any reasons why you split the tests in half and have this here? imho it should be all under es.rs

Reply:
Repeated from : https://github.com/Ballasi/num2words/pull/29#discussion_r1565132080
```
two actually.
1-) I wanted to keep the assert line into 1 line if I could
2-) It was a bit easier to see what changed between one line and the other. (ngl, the Closure was overkill just to skip 1 parameter but welp...)
```